excalidraw-app/collab/RoomDialog.tsx

```diff
@@ -64,12 +64,11 @@ export const RoomModal = ({
 
   const copyRoomLink = async () => {
     try {
-      await copyTextToSystemClipboard(
-        activeRoomLink,
-        null,
-        t("errors.copyToSystemClipboardFailed"),
-      );
-
+      try {
```

Comment:
let's remove the outer try/catch as it doesn't serve any purpose at the moment. Applies to the other 2 places that used to have a try/catch as well.